src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -1470,18 +1470,25 @@ private bool IsXamlTypeConverter(INamedTypeSymbol symbol)
 		private string BuildXamlTypeConverterLiteralValue(INamedTypeSymbol symbol, string memberValue, bool includeQuotations)
 		{
 			var attributeData = symbol.FindAttribute(XamlConstants.Types.CreateFromStringAttribute);
-			var returnType = attributeData?.NamedArguments.FirstOrDefault(kvp => kvp.Key == "MethodName").Value.Value;
+			var returnType = attributeData?.NamedArguments.FirstOrDefault(kvp => kvp.Key == "MethodName").Value.Value.ToString();
+
+			// Since the MethodName value can simply be the name of the method
+			// without its full path, example: nameof(MyConversionMethod), we must
+			// make sure to fully qualify the method name with its namespace
+			var fullyQualifiedReturnType = !returnType.Contains('.')
+				? $"{symbol.GetFullName()}.{returnType}"
```

Comment:
We'll also need to add a `global::` in front if the type is fully qualified

Reply:
done👍